src/site/topic16.rst

```diff
@@ -42,6 +42,7 @@ Unchecked
 
 * Unchecked exceptions are ``RuntimeException``s
 * These are exceptions that can be thrown during the normal operation of the JVM
+* These unchecked exceptions are not type checked by Java
```

Comment:
Maybe clearer, unchecked exceptions don't contribute the to type of the function (ie, `A => B throws E` doesn't get the `throws` if `E` inherits from `RuntimeException`).

Reply:
I'll add it for sure, but will perhaps do the discussion on the types during lecture and won't worry about going into the details in the contend.